app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
@josephelias-rubyians  Hi, lets try a rspec test for this request.

Reply:
Sure will do the Rspec. But before that I need to implement the validations for the fields like email, password etc.
Will do the Rspec once the validations is done.